cura/OAuth2/KeyringAttribute.py

```diff
@@ -39,6 +40,13 @@ def __get__(self, instance: "BaseModel", owner: type) -> Optional[str]:
                 self._store_secure = False
                 Logger.logException("w", "No keyring backend present")
                 return getattr(instance, self._name)
+            except Exception as e:
+                self._store_secure = False
+                exception_message = f"Something went wrong while trying to retrieve the password from the Keyring. Exception: {e}"
+                if Platform.isOSX() and hasattr(sys, "frozen") and type(e) == KeychainDenied:
+                    exception_message = "Access to the keyring was denied."
```

Comment:
In this case it probably shouldn't log a warning. The user explicitly denied access, so it is not a recoverable problem, but intentional. More like info?

Reply:
I changed it to log info in this case.